scripts/prepare.js

```diff
@@ -0,0 +1,36 @@
+const { spawn } = require('child_process')
+
+function runCommand(command, args) {
+  return new Promise((resolve, reject) => {
+    const process = spawn(command, args)
+
+    // Log when the process starts
+    console.log(`${command} ${args} initiated`)
+
+    // Uncomment to see output of each command
+    // process.stdout.on('data', (data) => {
+    //   console.log(`${command}: ${data}`)
+    // })
+
+    // Log errors
+    process.stderr.on('data', (data) => {
+      console.error(`${command} ${args} error: ${data}`)
+    })
+
+    // Log when and how the command exited
+    process.on('close', (code) => {
+      if (code !== 0) {
+        return reject(new Error(`${command} ${args} exited with code ${code}`))
+      }
+      console.log(`${command} ${args} completed successfully`)
```

Comment:
I think yarn will itself log when a command is completed. How much of the yarn command log is duplicated here? Do you even need to log when a command is initiated?

Reply:
I think the init log is useful to remind you what is executing under the hood.

Yarn doesn't log on its own when these child processes complete individually.